content/blog/2024/09/kaoto-release-2.2.0/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Kaoto v2.2 release"
+date: 2024-09-20
+draft: false
+authors: [lordrip]
+categories: ["Releases", "Tooling", "Kaoto"]
+preview: "Kaoto 2.2 has been released"
+---
+
+<div style="text-align: center">
+  <img src="cover.png" alt="Kaoto" height="400"/>
+</div>
+
+<br/>
+Following the Apache Camel 4.8 release, we are happy to announce the release of Kaoto 2.2 with many improvements to enhance the user experience.
+
+This time, the theme was to improve the visualizations and the user experience in the Canvas, as well as to provide more flexibility to the users. Read on to discover the key highlights of this release and how they can improve your workflow with Kaoto.
+
+## What’s New in Kaoto 2.2?
+
+### Canvas Improvements
+
+- New look and feel: The flow elements have been redesigned to provide a more modern and clean look. The new design is more intuitive and user-friendly, making it easier to work with your integrations.
+
+<div style="text-align: center">
+  <img src="canvas.png" alt="Canvas" height="400"/>
+</div>
+
+- Added containers for better organization: This feature allows you to expand and collapse integration parts, making it easier to work with complex integrations.
+
+<div style="text-align: center">
+  <img src="containersForBranches.png" alt="Containers for branches" height="400"/>
+</div>
+
+- Open source code to the side: You can now open the source code of your integration in a side panel, making it easier to work with both the visual and textual representation of your integration.
```

Comment:
it was always possible. The difference is that there are an editor quick action, a command and a keybinding to do it now faster.

Reply:
Right, I updated the text